Tests/Unit/NetworkServer/JsonHandlers/LnsStationConfigurationTests.cs

```diff
@@ -580,5 +580,110 @@ public void RegionConfigurationConverter_Throws_OnNotSetRegion()
             Assert.Equal(typeof(RegionAS923), region.GetType());
             Assert.Equal(exptectedOffset, ((RegionAS923)region).FrequencyOffset);
         }
+
+        [Fact]
+        public void RegionConfigurationConverter_CorrectlyReadsAS923Region_WithOffset_WithoutMutatingExistingOnes()
+        {
+            var input = @"{{
+            'msgtype': 'router_config',
+            'NetID': [1],
+            'JoinEui': [[0, 18446744073709551615]],
+	        'region': 'AS923',
+	        'hwspec': 'sx1301/1',
+	        'freq_range': [ 920000000, 925000000 ],
+            'DRs': [ [ 11, 125, 0 ],
+                     [ 10, 125, 0 ],
+                     [ 9, 125, 0 ],
+                     [ 8, 125, 0 ],
+                     [ 7, 125, 0 ],
+                     [ 7, 250, 0 ] ],
+            'sx1301_conf': [
+                {{
+                    'radio_0': {{
+                        'enable': true,
+                        'type': 'SX1257',
+                        'freq': {0},
```

Comment:
[nit] we might want to think about integrating this into the helper methods that we already have to replace tokens in that template. In that case, if the format changes, we don't have to fix it all over the place but just in a single place.

Reply:
Even though I agree with you, I think we should not do this as part of this PR.

Could you please discuss the idea tomorrow in the daily and eventually create a new issue, @bastbu? Thanks 👍